cmd/cloud/workspace.go

```diff
@@ -71,6 +147,47 @@ func workspaceSwitch(cmd *cobra.Command, out io.Writer, args []string) error {
 	return workspace.Switch(id, astroClient, out)
 }
 
+func addWorkspaceUsers(cmd *cobra.Command, args []string, out io.Writer) error {
+	var email string
+
+	// if an email was provided in the args we use it
+	if len(args) > 0 {
+		email = args[0]
+	}
+
+	cmd.SilenceUsage = true
+	return user.AddWorkspaceUser(email, role, "", out, astroCoreClient)
+}
+
+func listWorkspaceUsers(cmd *cobra.Command, out io.Writer) error {
+	cmd.SilenceUsage = true
+	return user.ListWorkspaceUsers(out, astroCoreClient, "")
+}
+
+func updateWorkspaceUsers(cmd *cobra.Command, args []string, out io.Writer) error {
+	var email string
+
+	// if an email was provided in the args we use it
+	if len(args) > 0 {
+		email = args[0]
+	}
```

Comment:
Do we need to throw an error for else case. in a scenario where they do not pass email address?

Reply:
It will ask you to pick from a list of users if an email is not provided. Like in deployment commands